src/_main.yml

```diff
@@ -86,8 +86,11 @@ repository:
       "2":
         patterns:
           - name: variable.other.enummember.hcl
-            comment: Block label
-            match: '[\"\-\w]+'
+            comment: Block label (String Literal)
+            match: '\"[^\"\r\n]*\"'
+          - name: variable.other.enummember.hcl
+            comment: Block label (Indentifier)
+            match: '[[:alpha:]][[:alnum:]_-]*'
```

Comment:
```suggestion
            match: '[[:alpha:]][[:alnum:]_-]+'
```

Reply:
No, this needs to be `*`.  Otherwise you can't have single letter identifiers. That pattern was taken from https://github.com/hashicorp/syntax/blob/main/src/_main.yml#L23

Reply:
Interesting ... I'd have thought that `+` still accounts for a single letter by definition but I guess there is the context of other regexes.

In that case would you mind adding a test for such a single letter identifier, which will fail on `*`? 🙏🏻

Reply:
> I'd have thought that + still accounts for a single letter by definition
It does but the previous `[[:alpha:]]` clause requires a single alpha character so you end up with;

(requires only 1 char)(requires 1 or more chars) 

which means it requires 2 or more chars.

I'll add a test for that too. Easy enough.

Reply:
Test added.